configs/dc2.yaml

```diff
@@ -0,0 +1,51 @@
+facility_name: Rubin-LSST
+obs_collection: LSST.DC2
+collections: ["2.2i/runs/DP0.1"]
+use_butler_uri: false
+dataset_types:
+  raw:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.raw
+    calib_level: 1
+    obs_id_fmt: "{records[exposure].obs_id}"
+    o_ucd: phot.count
+    access_format: image/fits
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+  calexp:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.calexp
+    calib_level: 2
+    obs_id_fmt: "{records[visit].name}"
+    o_ucd: phot.count
+    access_format: image/fits
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+  deepCoadd_calexp:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.coadd
+    calib_level: 3
+    obs_id_fmt: "{skymap}-{tract}-{patch}"
+    o_ucd: phot.count
+    access_format: image/fits
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+  deepCoadd_forced_src:
+    dataproduct_type: measurements
+    dataproduct_subtype: lsst.forced_src
+    calib_level: 3
+    obs_id_fmt: "{skymap}-{tract}-{patch}"
+    access_format: application/x-votable+xml;content=datalink
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+spectral_ranges:
+  "HSC-G": [400., 500.]
+  "HSC-R": [500., 700.]
+  "HSC-R2": [500., 700.]
+  "HSC-I": [700., 850.]
+  "HSC-I2": [700., 850.]
+  "HSC-Y": [950, 1000]
+  "HSC-Z": [900, 950]
+  "N921": [500., 700.]
+  "u": [300., 400.]
+  "g": [400., 500.]
+  "r": [500., 700.]
+  "i": [700., 850.]
+  "z": [950, 1000]
+  "y": [900, 950]
```

Comment:
I think "z" and "y" got swapped here.

Reply:
They are all  more or less random numbers, but I'll try to find better set.